src/services/aragon-sdk/queries/use-member.ts

```diff
@@ -80,13 +80,15 @@ export const useMember = (
   options: UseQueryOptions<TokenVotingMember> = {}
 ) => {
   const client = usePluginClient('token-voting.plugin.dao.eth');
-  const {address, network} = useWallet();
+  const {address} = useWallet();
+  const {network} = useNetwork();
+
   const baseParams = {
     address: address as string,
```

Comment:
We don't probably need the current user address to build the query key, I would suggest to remove the `useWallet` hook here:
```suggestion
  const {network} = useNetwork();

  const baseParams = {
```
And update the `getMember` query key params to omit the `address` parameter as follow:
```
getMember: (
    baseParams: Omit<IAragonSdkBaseParams, 'address'>,
    params: IFetchMemberParams
```